.env.example

```
@@ -70,6 +70,22 @@ SC_MONGODB_URI=""
+# SAFE_PROPOSAL_REASON="whitelist sync stage 4c" bun run propose-safe-tx
+SAFE_PROPOSAL_REASON=
+
+# Set to "bot" when an unattended job creates Safe proposals, so provenance
+# records the actor as a bot instead of whichever git identity the host has
+# configured. Leave empty for humans and CI (CI is detected automatically).
+SAFE_PROPOSAL_ACTOR=
+# Escape hatch for confirm-safe-tx: when "true", a proposal whose nonce is ahead
+# of the Safe's on-chain nonce may still be broadcast. Default OFF — such a
+# broadcast reverts with GS026. Set it only when the configured RPC is known to
+# report an out-of-date on-chain nonce.
+ALLOW_FUTURE_NONCE_EXECUTION=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)